docs/src/modules/ROOT/pages/constraints-and-score/score-calculation.adoc

```diff
@@ -1504,113 +1476,18 @@ public interface IncrementalScoreCalculator<Solution_, Score_ extends Score<Scor
 
 image::constraints-and-score/score-calculation/incrementalScoreCalculatorSequenceDiagram.png[align="center"]
 
-For example in n queens:
-
-[source,java,options="nowrap"]
-----
-public class NQueensAdvancedIncrementalScoreCalculator
```

Comment:
Does it make sense to add the incremental score calculator?

Reply:
Decided not to. People should be writing constraint providers; incremental is still there, but let's not draw attention to it.